Networking/Networking/Model/Stats/ProductsReportItem.swift

```diff
@@ -44,19 +44,25 @@ public struct ProductsReportItem: Decodable, Equatable, GeneratedFakeable {
         let quantity = try container.decode(Int.self, forKey: .quantity)
         let total = try container.decode(Double.self, forKey: .total)
 
-        let extendedInfo = try container.nestedContainer(keyedBy: ExtendedInfoCodingKeys.self, forKey: .extendedInfo)
-
-        let productName = try extendedInfo.decode(String.self, forKey: .productName)
-
-        // Parse and extract the `src` string out of the image html using `Aztec parser`
-        let imageUrl: String? = {
-            guard let imageHTML = try? extendedInfo.decodeIfPresent(String.self, forKey: .imageHTML),
-            let img = HTMLParser().parse(imageHTML).firstChild(ofType: .img),
-            let src = img.attribute(ofType: .src)?.value.toString() else {
-                return nil
-            }
-            return src
-        }()
+        let productName: String
+        let imageUrl: String?
+        if let simplyDecodedProductName = try? container.decode(String.self, forKey: .productName) {
```

Comment:
This is a quick solution, as now we can have two different sources to decode it (json from networking and the cached one) we have to handle both ways. We can come up with a cleaner solution on this PR next iteration.